docs/tutorials/conntrack.rst

```diff
@@ -0,0 +1,353 @@
+            - rule:
+                eth_type: 0x0800 # ipv4
+                ct_state: 0/0x20  # match -trk (untracked)
+                actions:
+                    # Re-inject the tracked packet into the OpenFlow pipeline, containing
+                    # additional connection metadata, to default table 0. The tracked packet
+                    # is again evaluated by Faucet ACLs in table 0. The original, untracked
+                    # packet is effectively dropped.
+                    ct:
+                        zone: 10 # arbitrary conntrack zone ID to match against later
+                        table: 0
+            # Commit NEW IPv4 connections from host1 to host2
+            - rule:
+                eth_type: 0x0800 # ipv4
+                ipv4_src: 10.0.0.1
+                ipv4_dst: 10.0.1.2
+                ct_state: 0x21/0x21 # match +new - packets to establish a new connection
+                actions:
+                    # Commit the connection to the connection tracking module which will be
+                    # stored beyond the lifetime of packet in the pipeline.
+                    ct:
+                        zone: 10  # the same conntrack zone ID as above
+                        flags: 1  # "commit" the new connection
+                        table: 1  # implicit "allow" new connection packet(s) via faucet table 1
+            # Allow packets in either direction from existing connections initiated by
+            # host1 only
+            - rule:
+                eth_type: 0x0800 # ipv4
+                ct_zone: 10 # match packets associated with our conntrack zone ID
+                ct_state: 0x22/0x22 # match +est - packets in an established connection
+                actions:
+                    allow: True
+            # Block all unwanted packets and new connections from host2 to host1
+            - rule:
+                eth_type: 0x0800 # ipv4
+                ipv4_src: 10.0.1.2
+                ipv4_dst: 10.0.0.1
+                actions:
+                    allow: False
+
+Be sure to also apply the new ACL to both ports in the data plane.
+
+.. code-block:: yaml
+    :caption: /etc/faucet/faucet.yaml
+
+    dps:
+        sw1:
+            1:
+                ...
+                acls_in:
+                - conntrack_fw
+            2:
+                ...
+                acls_in:
+                - conntrack_fw
+
+Reload Faucet to apply the new configuration.
+
+.. code:: console
+
+    sudo systemctl reload faucet
+
+The new conntrack related ACLs should have been added:
+
+.. code:: console
+
+    ovs-ofctl dump-flows br0 -O OpenFlow13 | grep =ct
+
+We can debug how OVS interfaces with the conntrack module to deal with the tracked
+packet(s):
+
+.. code:: console
+
+    ovs-appctl ofproto/trace br0 in_port=1,tcp,nw_src=10.0.0.1,nw_dst=10.0.1.2
+
+Our ping from host1 to host2 should continue to work, establishing an entry in the
+connection tracker. 
```

Comment:
Suggest adding a call to the `conntrack` command after the ping to demonstrate the entry has been created, the output would look like this:

```
$ sudo conntrack -L | grep 10.0.1.2
conntrack v1.4.5 (conntrack-tools): 19 flow entries have been shown.
icmp     1 26 src=10.0.0.1 dst=10.0.1.2 type=8 code=0 id=11240 src=10.0.1.2 dst=10.0.0.1 type=0 code=0 id=11240 mark=0 zone=10 use=
```

It would also need to be necessary to add a prerequisite to the top section of the tutorial for the user to install the conntrack as it's not a default (e.g. `apt-get install conntrack`)